weblate/addons/management/commands/install_addon.py

```diff
@@ -67,7 +67,8 @@ def handle(self, *args, **options):
             addons = Addon.objects.filter_component(component).filter(name=addon.name)
             if addons.exists():
                 if options["update"]:
-                    addons.update(configuration=configuration)
+                    addon_component = addon.__class__(addons[0])
+                    addon_component.configure(configuration)
```

Comment:
This looks wrong as it calls the `configure` method only for first addon, while the previous code did update it for all of them.

```suggestion
                    for addon_component in addons:
                        addon_component.configure(configuration)
```

Reply:
But there can be only one addon per component